interface.py

```diff
@@ -92,11 +96,12 @@ def switch_power_down(self):
                 self.power_down = True
             else:
                 self.power_down = False
+        self.change_callback()
 
     def confirm_selection(self):
         """
         When indicator is False the player can choose cards and switch Power Down.
         When is True the player ended the selection of cards.
         """
-        if None not in self.program:
-            self.selection_confirmed = True
+        self.selection_confirmed = True
```

Comment:
Zde mazes logiku potvrzeni karet. Da se potvrdit interface, i kdyz sis nevybrala zadnou kartu. Predpokladam, ze se jedna o bug?

Reply:
jojo já tu měla myslím conflict z masteru a špatně jsem to udělala, díky.